llvm/plan.sh

```diff
@@ -32,44 +32,78 @@ do_unpack() {
   # There may be some more awesome way to do this - I don't know that yet.
   build_line "Unpacking $pkg_filename to custom cache dir"
   local source_file=$HAB_CACHE_SRC_PATH/$pkg_filename
-  local unpack_dir=$HAB_CACHE_SRC_PATH/${pkg_name}-${pkg_version}
+  # we want to keep the src as some later dependencies require it
+  local unpack_dir="${pkg_prefix}/src"
   mkdir -p "$unpack_dir"
-  pushd "$unpack_dir" > /dev/null
+  pushd "$unpack_dir" > /dev/null || exit
```

Comment:
Why all the calls to `exit`? iirc the build program should die if any of these fail

Reply:
If I remember correctly, it was because shellchecked failed on this https://github.com/koalaman/shellcheck/wiki/SC2164. `pushd` failed like `cd`.